2.7.0/final.md

```diff
@@ -0,0 +1,958 @@
+# PyTorch 2.7.0 Release Notes
+- [Highlights](#highlights)
+- [Backwards Incompatible Changes](#backwards-incompatible-changes)
+- [Deprecations](#deprecations)
+- [New Features](#new-features)
+- [Improvements](#improvements)
+- [Bug fixes](#bug-fixes)
+- [Performance](#performance)
+- [Documentation](#documentation)
+- [Developers](#developers)
+- [Security](#security)
+
+# Highlights
+TODO @seemethere
+
+For more details about these highlighted features, you can look at the release blogpost.
+Below are the full release notes for this release.
+
+# Backwards Incompatible Changes
```

Comment:
I would suggest doing a final re-ordering pass on the content here to have at the top what you expect the largest number of people will want to know about !